flow/packet-headers/ipv4.yaml

```diff
@@ -403,3 +409,19 @@ components:
             default: 0
             features: [count, metric_tags]
           x-field-uid: 6
+    Flow.Ipv4.Auto:
+      description: |-
+        The OTG implementation can provide a system generated, value for this property.
+      type: object
+      required: [choice]
+      properties:
+        choice:
+          description: |- 
+            The method to be used to provide the system generated value.
+            
+            dhcp option populates this property based on the dynamic ip that is been allocated by the dhcp server.
```

Comment:
Maybe little better might be on the lines : 
" The dhcp option populates the field based on the dynamic IPv4 address that  has been assigned to the DHCPv4 client by a DHCPv4 server."

Reply:
made the necessary changes